### ISSUE TITLE
[Tablet Orders] Adding products when creating a new order causes -sometimes- a view glitch

### ISSUE THREAD

**Issue by iamgabrielma** [woocommerce/woocommerce-ios#11766]
For some reason this doesn't show in the recording despite I can clearly replicate and see it on the physical device, but it does show if I make an screenshot (?)

To reproduce:
~~1. On a physical device, tap on the "scan" button to scan a product via barcode scanner to add it to the order~~
2. Once added, tap in `+` to add a product from the list 
3. Observe how the order creation view is not correctly shown. Tapping on the products to the left, or the bottom drawer fixes this state.

![IMG_0601](https://github.com/woocommerce/woocommerce-ios/assets/3812076/cde59336-ca2d-4d8a-ae04-64285d47a054)

**Comment by joshheald** [comment]
I can't repro this; please could you check with `trunk` now that everything's merged?

**Comment by iamgabrielma** [comment]
Yes, this is in trunk. On an iPad Pro 12.9-inch 3rd gen, OS 15.3.1 . I've updated the title since it also happens when I don't scan products, but also when I just add products manually (It didn't happen the first times though, so is not always) 🤔 

I'm not quite sure how to debug this further, this is the view hierarchy:

<img width="1470" alt="Screenshot 2024-01-23 at 20 08 03" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/0b383b64-cc2d-4651-9972-555e90fae81b">

<img width="1453" alt="Screenshot 2024-01-23 at 20 08 11" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/615c2ca6-f66f-4511-9498-c2cc6d10c731">

From my understanding could be pointing to an issue between the [ScrollView and the grouped VStacks here](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/ViewRelated/Orders/Order%20Creation/OrderForm.swift#L132-L136). I'm wondering if could be related to some combination of the use of `ScrollViewProxy` and `.ignoresSafeArea` horizontally on that view?

**Comment by joshheald** [comment]
Yep, I can repro this on iOS 15 too. Not got time to dig in to it immediately but I'll look later. Just wanted to confirm it's not only you.

I can repro it on an iPhone in Portrait too, so it's not limited to iPad.

I can also repro it on WCiOS 16.5 (from TestFlight), which predates all the work we've done here. It's a long-standing issue.

**Comment by joshheald** [comment]
Automated triage note: this was tested and commented by Codex using device/simulator automation. Partially retested on iPad Air (M1) running iPadOS 18.7.1, Woo (Dev) 24.9 (24.9.0.4) at df01963e. I opened a new order draft and selected one product, then a second product, from the product list. The split order creation view stayed coherent/tappable in this pass, so I could not reproduce a separate malformed-view glitch. I did see the duplicate Recalculate controls, but recorded that separately under #12224 because it matches that issue more directly.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by iamgabrielma** [comment]
Closing as part of migrating mobile issue tracking from GitHub to Linear. This intermittent rendering glitch was last confirmed on iOS 15-era builds and could not be reproduced in a 2026 retest on iPadOS 18.7. Not being carried forward to the Linear backlog; if it resurfaces on current OS versions it can be re-filed with a fresh repro.

_This action was performed by an automated triage process run by @iamgabrielma, as part of migrating mobile issue tracking from GitHub to Linear. If this was closed in error, please reopen._
